core/src/main/java/com/epam/esm/entities/Certificate.java

```diff
@@ -0,0 +1,117 @@
+package com.epam.esm.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Stores data from "gift_certificate" table
+ */
+public class Certificate extends Entity {
+    private static final long serialVersionUID = 128L;
+
+    private List<Tag> tagList;
+    private String name;
+    private String description;
+    private Integer price;
```

Comment:
@KukharchukDaniil We need use BigDecemal for this field.

Reply:
fixed